Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (11.0.1)
-    coderay (1.1.2)
-    commander (4.4.7)
+    byebug (11.1.3)
+    coderay (1.1.3)
+    commander (4.5.2)
       highline (~> 2.0.0)
-    database_cleaner (1.7.0)
-    diff-lcs (1.3)
+    database_cleaner (1.8.5)
+    diff-lcs (1.4.2)
     erubi (1.9.0)
     event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
     highline (2.0.3)
-    method_source (0.9.2)
-    mustermann (1.0.3)
+    method_source (1.0.0)
+    mustermann (1.1.1)
+      ruby2_keywords (~> 0.0.1)
     pg (0.20.0)
-    pry (0.12.2)
-      coderay (~> 1.1.0)
-      method_source (~> 0.9.0)
-    pry-byebug (3.7.0)
+    pry (0.13.1)
```

Comment:
### [pry](http://pry.github.io)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.12.2 → 0.13.1

[_[change-log](https://github.com/pry/pry/blob/master/CHANGELOG.md), [source-code](https://github.com/pry/pry)_]





<details>
<summary>Commits</summary>

A change of **492** commits. See the full changes on [the compare page](https://github.com/pry/pry/compare/v0.12.2...v0.13.1).

These are the first 10 commits:
- (f0ec14c) [Configure SimpleCov properly](https://github.com/pry/pry/commit/f0ec14c5c71da182214e63825dfa88bf8fa98fda)
- (175423f) [circle: update Dockerfiles to have make & gcc installed](https://github.com/pry/pry/commit/175423ff9d4977969234eaf050239431ff4bdded)
- (ec63172) [Merge pull request #1970 from pry/docker-make](https://github.com/pry/pry/commit/ec63172bf73de6da062c92580e375aacf2ca8e21)
- (bf3d0bf) [Merge pull request #1969 from pry/simplecov-fix](https://github.com/pry/pry/commit/bf3d0bffa0633a12a1aeaf2f1bd6768fe3342eaf)
- (51ec2f7) [Move Pry::DEFAULT_HOOKS to Config](https://github.com/pry/pry/commit/51ec2f70e68bafa7dc1feb876f691cf4e8d587ba)
- (4e3200e) [Merge pull request #1971 from pry/default-hooks-moving](https://github.com/pry/pry/commit/4e3200eececbc2949de0de577b3deae432b28414)
- (19db442) [Move Pry::DEFAULT_PRINT to Config](https://github.com/pry/pry/commit/19db4422634b3c382dbd4168b3f244c3448921ea)
- (20b7d04) [Move Pry::SIMPLE_PRINT and CLIPPED_PRINT to Inspector](https://github.com/pry/pry/commit/20b7d049527af793590139729b959a28646c5703)
- (acdcd72) [Move Pry::DEFAULT_EXCEPTION_HANDLER to Config](https://github.com/pry/pry/commit/acdcd720dae99497bd137946491725792681b138)
- (2e25901) [Move Pry::DEFAULT_CONTROL_D_HANDLER to Config](https://github.com/pry/pry/commit/2e259018d3f9678aaf5134d9d5672a119b202305)


</details>